src/spikeinterface/extractors/neoextractors/spikegadgets.py

```diff
@@ -36,6 +38,11 @@ def __init__(self, file_path, stream_id=None, stream_name=None, block_index=None
         )
         self._kwargs.update(dict(file_path=str(Path(file_path).absolute()), stream_id=stream_id))
 
+        probegroup = probeinterface.read_spikegadgets(file_path)
```

Comment:
It should handle the case when `read_spikegadgets` returns None (as it should in case of no probe found)